README.md

```diff
@@ -52,23 +52,68 @@ docker run cisagov/domain-manager-ui:1.0.0
 1. Start the container and detach:
 
     ```console
-    docker-compose up --detach
+    docker compose up --detach
     ```
 
+<<<<<<< HEAD
+=======
+## Using secrets with your container ##
+
+This container also supports passing sensitive values via [Docker
+secrets](https://docs.docker.com/engine/swarm/secrets/).  Passing sensitive
+values like your credentials can be more secure using secrets than using
+environment variables.  See the
+[secrets](#secrets) section below for a table of all supported secret files.
+
+1. To use secrets, create a `quote.txt` file containing the values you want set:
+
+    ```text
+    Better lock it in your pocket.
+    ```
+
+1. Then add the secret to your `docker-compose.yml` file:
+
+    ```yaml
+    ---
+    version: "3.7"
+
+    secrets:
+      quote_txt:
+        file: quote.txt
+
+    services:
+      example:
+        image: cisagov/example:0.0.1
+        volumes:
+          - type: bind
+            source: <your_log_dir>
+            target: /var/log
+        environment:
+          - ECHO_MESSAGE="Hello from docker compose"
+        ports:
+          - target: 8080
+            published: 8080
+            protocol: tcp
+        secrets:
+          - source: quote_txt
+            target: quote.txt
+    ```
+
+>>>>>>> 9cea64da4a1379cdde3c509a09fc548f0da8b034
```

Comment:
You have committed a merge conflict. Please resolve.

Reply:
resolved with [d38e133](https://github.com/cisagov/domain-manager-ui/pull/247/commits/d38e1335f747335baa33f87d2745c9cf0cc84808)